tests/model_inheritance/test_abstract_inheritance.py

```diff
@@ -0,0 +1,376 @@
+from __future__ import unicode_literals
+
+from django.apps import apps
+from django.contrib.contenttypes.fields import (
+    GenericForeignKey, GenericRelation,
+)
+from django.contrib.contenttypes.models import ContentType
+from django.core.checks import Error
+from django.core.exceptions import FieldDoesNotExist, FieldError
+from django.db import models
+from django.test import TestCase
+
+
+class IsolatedModelsTestCase(TestCase):
```

Comment:
This should be replaced with `@isolate_apps()` as done in a08fda2111d811aa53f11218fa03f3300dfff4cb.